build-scripts/install_tensorflow.sh

```diff
@@ -0,0 +1,8 @@
+mkdir -p /root/src/tflite
+cd /root/src/tflite
+
+if [ ! -d "tensorflow_src" ] 
```

Comment:
Normally we name the folder with `-` instead of `_`, how about putting this file under `<wamr_root>/core/deps` with content:
```bash
mkdir -p tflite
cd tflite

if [ ! -d "tensorflow-src" ]
then
    git clone https://github.com/tensorflow/tensorflow.git tensorflow-src \
        --branch v2.10.0
fi
```